core/modules/installer/installer.module

```diff
@@ -0,0 +1,516 @@
+<?php
+/**
+ * @file
+ * Handles installation and updates of contributed projects.
+ */
+
+/**
+ * Implements hook_permission().
+ */
+function installer_permission() {
+  return array(
+    'use installer browser' => array(
+      'title' => t('Access installer interface'),
+      'description' => t('This allows the user to browse for and install new modules, themes and layouts using Installer Browser.'),
```

Comment:
We can remove `using Installer Browser` here. We don't need to name it, and naming things is hard :)